app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -1,343 +1,320 @@
-  $ sudo mv bintray-kong-kong-enterprise-edition-aws.repo /etc/yum.repos.d/
-  ```
-
-3. Open the package to and add your Bintray credentials to the `baseurl` value (See [Getting your Bintray Credentials] for help)
-  ```
-  $ sudo vi /etc/yum.repos.d/bintray-kong-kong-enterprise-edition-aws.repo
-  ```
-  ```
-  baseurl=https://<BINTRAY_USER>:<BINTRAY_API_KEY>@kong.bintray.com/kong-enterprise-edition-aws
-  ```
-    > Note: Unlike step 1, this command requires your Bintray account API KEY, **not** your PASSWORD.
-
-4. Obtain your Kong Enterprise license file.
-  If you do not already have your license file, you can download it from your account files in Bintray *`https://bintray.com/kong/<YOUR_REPO_NAME>/license#files`*. See [How you Access Your Enterprise License](/enterprise/latest/deployment/access-license) for help.
+To complete this guide you will need:
 
-5. Ensure your license file is in proper *`JSON`*:
-  ```
-    {"license":{"signature":"91e6dd9716d12ffsn4a5ckkb16a556dbebdbc4d0a66d9b2c53f8c8d717eb93dd2bdbe2cb3ef51c20806f14345128907da35","payload":{"customer":"Kong Inc","license_creation_date":"2019-05-07","product_subscription":"Kong Enterprise Edition","admin_seats":"5","support_plan":"None","license_expiration_date":"2021-04-01","license_key":"00Q1K00000zuUAwUAM_a1V1K000005kRhuUAE"},"version":1}}
-  ```
+* A valid Bintray account. You will need your **username**, account **password** and account **API Key**.
+  * Example:
+    * **Bintray Access key**: `john-company`
+    * **Bintray username**: `john-company@kong`
+    * **Bintray password**: `12345678`
+    * **Bintray API key**: `12234e314356291a2b11058591bba195830`
+  * The API Key can be obtainedby visiting [https://bintray.com/profile/edit](https://bintray.com/profile/edit) and selecting **API Key**
+* A supported Amazon Linux 2 system with root equivalent access.
+* A valid Kong Enterprise license JSON file, this can be found in your Bintray account. See [Accessing Your License](/enterprise/latest/deployment/access-license)
 
-6. Securely copy the license file to the EC2 instance
-  ```
-    $ scp license.json <ec2-username>@<serverip>:~
-  ```
+## Step 1. Prepare to Install Kong Enterprise and Download the License File
 
+There are two options to install Kong Enterprise on Amazon Linux 2. Both will require a login to Bintray.
 
-## Step 2. Install Kong Enterprise
+Log in to [Bintray](http://bintray.com). Your Kong Sales or Support contact will assign credentials to you.
 
-1. Install Kong Enterprise
-  ```
-  $ sudo yum install kong-enterprise-edition
-  ```
+### Option 1: Download RPM File
+   
+1. Go to: https://bintray.com/kong/kong-enterprise-edition-aws. 
+2. Select the `aws` folder. Kong Enterprise versions are listed in reverse chronological order.
+3. Select the latest Kong version from the list.
+4. From the Kong version detail page, select the **Files** tab and click the dist folder. 
+5. Save the RPM file available: e.g. `kong-enterprise-edition-1.3.0.1.aws.rpm`
```

Comment:
Change  "Save the RPM file available: e.g. `kong-enterprise-edition-1.3.0.1.aws.rpm`" to 
"Save the RPM file. For example,`kong-enterprise-edition-1.3.0.1.aws.rpm"